src/components/Footer/Footer.scss

```
@@ -0,0 +1,55 @@
+.footer-area {
+  display: flex;
+  justify-content: space-around;
+  position: fixed;
+  left: 0px;
+  right: 0px;
+  bottom: 0px;
```

속성 값이 0 일 경우 단위를 표시하지 않는 게 일반적입니다.
```suggestion
  left: 0;
  right: 0;
  bottom: 0;
```